src/quo2/components/icons/icons.clj

```diff
@@ -4,9 +4,14 @@
 
 (def icon-path "./resources/images/icons2/")
 
+(defn combine-path [path el]
+  (if (System/getenv "__COMPONENT_TEST__")
```

Comment:
similar change here @flexsurfer, where the resources are not found in the test setup so I had to adjust the path a bit for when using tests mode.